runtime/src/iree/base/internal/tsan.h

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 The IREE Authors
```

Comment:
add these to debugging.h, which already has asan defines in it

Reply:
done